lib/editor/util/validation.js

```diff
@@ -313,6 +313,10 @@ export function validate (
           scheduleExceptions.splice(exceptionIndex, 1)
         }
         const castedScheduleException: ScheduleException = ((entity: any): ScheduleException)
+        if (scheduleExceptions.map(ex => ex.name).includes(castedScheduleException.name)) {
+          const reason = `${castedScheduleException.name} is already used in another exception`
```

Comment:
Since this is referring to the name of the exception, should it maybe have quotation marks around the name?

Reply:
Great idea! Added that in https://github.com/ibi-group/datatools-ui/pull/840/commits/6102b671a8a77dbdd5e242b0f9f9a2d6bc521b84